{
  "uid" : "5c2ae513dedd258e",
  "name" : "existing users",
  "fullName" : "authentication existing users",
  "historyId" : "2e7c1d7d-f0b1-505b-ace8-9918dedf46b0",
  "time" : {
    "start" : 1652357375106,
    "stop" : 1652357376595,
    "duration" : 1489
  },
  "description" : "When user is already signed up then he should be able to log in",
  "descriptionHtml" : "<p>When user is already signed up then he should be able to log in</p>\n",
  "status" : "failed",
  "statusMessage" : "PostgresError: password authentication failed for user \"postgres\"",
  "statusTrace" : "\n    at ErrorResponse (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:775:26)\n    at handle (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:484:6)\n    at Socket.data (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:328:9)\n    at Socket.emit (events.js:400:28)\n    at Socket.emit (domain.js:475:12)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)\n    at Socket.Readable.push (internal/streams/readable.js:206:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)\n    at cachedError (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/query.js:160:23)\n    at new Query (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/query.js:36:24)\n    at Function.sql (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/index.js:104:11)\n    at Authentication.<anonymous> (/home/runner/work/infrastructure/infrastructure/supabase/tests/features/javascript/hooks.ts:77:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "description" : "When user is already signed up then he should be able to log in",
    "status" : "failed",
    "statusMessage" : "PostgresError: password authentication failed for user \"postgres\"",
    "statusTrace" : "\n    at ErrorResponse (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:775:26)\n    at handle (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:484:6)\n    at Socket.data (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:328:9)\n    at Socket.emit (events.js:400:28)\n    at Socket.emit (domain.js:475:12)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)\n    at Socket.Readable.push (internal/streams/readable.js:206:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)\n    at cachedError (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/query.js:160:23)\n    at new Query (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/query.js:36:24)\n    at Function.sql (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/index.js:104:11)\n    at Authentication.<anonymous> (/home/runner/work/infrastructure/infrastructure/supabase/tests/features/javascript/hooks.ts:77:21)",
    "steps" : [ {
      "name" : "Create a valid user",
      "time" : {
        "start" : 1652357375107,
        "stop" : 1652357376595,
        "duration" : 1488
      },
      "status" : "failed",
      "statusMessage" : "password authentication failed for user \"postgres\"",
      "statusTrace" : "PostgresError: password authentication failed for user \"postgres\"\n    at ErrorResponse (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:775:26)\n    at handle (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:484:6)\n    at Socket.data (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:328:9)\n    at Socket.emit (events.js:400:28)\n    at Socket.emit (domain.js:475:12)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)\n    at Socket.Readable.push (internal/streams/readable.js:206:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)\n    at cachedError (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/query.js:160:23)\n    at new Query (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/query.js:36:24)\n    at Function.sql (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/index.js:104:11)\n    at Authentication.<anonymous> (/home/runner/work/infrastructure/infrastructure/supabase/tests/features/javascript/hooks.ts:77:21)",
      "steps" : [ {
        "name" : "Create Supabase client",
        "time" : {
          "start" : 1652357375109,
          "stop" : 1652357375109,
          "duration" : 0
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : false,
        "hasContent" : false
      }, {
        "name" : "I sign up with a valid email and password",
        "time" : {
          "start" : 1652357375110,
          "stop" : 1652357376568,
          "duration" : 1458
        },
        "status" : "passed",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : false,
        "hasContent" : false
      }, {
        "name" : "get confirmation token for user 3c6594eb-02e1-47c5-9814-72d38337ec55",
        "time" : {
          "start" : 1652357376569,
          "stop" : 1652357376595,
          "duration" : 26
        },
        "status" : "failed",
        "statusMessage" : "password authentication failed for user \"postgres\"",
        "statusTrace" : "PostgresError: password authentication failed for user \"postgres\"\n    at ErrorResponse (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:775:26)\n    at handle (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:484:6)\n    at Socket.data (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/connection.js:328:9)\n    at Socket.emit (events.js:400:28)\n    at Socket.emit (domain.js:475:12)\n    at addChunk (internal/streams/readable.js:293:12)\n    at readableAddChunk (internal/streams/readable.js:267:9)\n    at Socket.Readable.push (internal/streams/readable.js:206:10)\n    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)\n    at cachedError (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/query.js:160:23)\n    at new Query (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/query.js:36:24)\n    at Function.sql (/home/runner/work/infrastructure/infrastructure/supabase/tests/node_modules/postgres/cjs/src/index.js:104:11)\n    at Authentication.<anonymous> (/home/runner/work/infrastructure/infrastructure/supabase/tests/features/javascript/hooks.ts:77:21)",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 3,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "thread",
    "value" : "01"
  }, {
    "name" : "package",
    "value" : "authentication"
  }, {
    "name" : "parentSuite",
    "value" : "authentication"
  }, {
    "name" : "severity",
    "value" : "blocker"
  }, {
    "name" : "feature",
    "value" : "authentication"
  }, {
    "name" : "framework",
    "value" : "jest"
  }, {
    "name" : "language",
    "value" : "typescript/javascript"
  }, {
    "name" : "host",
    "value" : "fv-az210-147"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "blocker",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 5,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 5
      },
      "items" : [ {
        "uid" : "fa57bd5a493d8289",
        "reportUrl" : "https://supabase.github.io/test-reports/verification-stage/8//#testresult/fa57bd5a493d8289",
        "status" : "failed",
        "statusDetails" : "PostgresError: password authentication failed for user \"postgres\"",
        "time" : {
          "start" : 1652356427763,
          "stop" : 1652356429563,
          "duration" : 1800
        }
      }, {
        "uid" : "835a190d2ef2e2e6",
        "reportUrl" : "https://supabase.github.io/test-reports/verification-stage/8//#testresult/835a190d2ef2e2e6",
        "status" : "failed",
        "statusDetails" : "PostgresError: password authentication failed for user \"postgres\"",
        "time" : {
          "start" : 1652355688009,
          "stop" : 1652355689350,
          "duration" : 1341
        }
      }, {
        "uid" : "23cdb5de8aeca9bc",
        "reportUrl" : "https://supabase.github.io/test-reports/verification-stage/7//#testresult/23cdb5de8aeca9bc",
        "status" : "failed",
        "statusDetails" : "Error: expect(received).toBeNull()\n\nReceived: [FetchError: request to http://localhost:8000/auth/v1/signup failed, reason: connect ECONNREFUSED 127.0.0.1:8000]",
        "time" : {
          "start" : 1652354650004,
          "stop" : 1652354650043,
          "duration" : 39
        }
      }, {
        "uid" : "1082c5584147088a",
        "reportUrl" : "https://supabase.github.io/test-reports/verification-stage/6//#testresult/1082c5584147088a",
        "status" : "failed",
        "statusDetails" : "Error: expect(received).toBeNull()\n\nReceived: [FetchError: request to http://localhost:8000/auth/v1/signup failed, reason: connect ECONNREFUSED 127.0.0.1:8000]",
        "time" : {
          "start" : 1652304992232,
          "stop" : 1652304992254,
          "duration" : 22
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "5c2ae513dedd258e.json",
  "parameterValues" : [ ]
}